packages/components/src/components/Dialog/Dialog.tsx

```diff
@@ -251,11 +277,17 @@ export const DialogFooter = (
 DialogFooter.displayName = 'Dialog.Footer';
 
 export const DialogBody = (
-    { children, 'data-test-id': dataTestId = 'fondue-dialog-body' }: DialogBodyProps,
+    { padding = 'compact', children, 'data-test-id': dataTestId = 'fondue-dialog-body' }: DialogBodyProps,
```

Comment:
if we set a default value here, wouldn't it will always override the dialog-wide padding?